packages/Adapters/STKMesh/CMakeLists.txt

```diff
@@ -15,6 +15,11 @@ TRIBITS_SUBPACKAGE(STKMeshAdapters)
 ## Add library, test, and examples.
 ##---------------------------------------------------------------------------##
 
+# STK adapters do not work with "-Werror -Wall"
+STRING(REPLACE "-Werror" "" CUSTOM_FLAGS ${CMAKE_CXX_FLAGS})
+SET(CMAKE_CXX_FLAGS "${CUSTOM_FLAGS}")
+
+
```

Comment:
A different approach would be to disable STK mesh adapters when running with `-Werror`.